src/app/login/login.component.scss

```diff
@@ -37,17 +37,75 @@
 
 .card-actions {
   margin-top: 20px;
+
+  .separator {
+    margin: 20px 0;
+    border-top: 1px solid #CCCCCC;
+    position: relative;
+
+    .separator-text {
+      display: block;
+      position: absolute;
+      top: -10px;
+      left: 50%;
+      margin-left: -22px;
+      padding: 0px 10px;
+      background: #FFFFFF;
+      color: #8a8a8a;
+    }
+  }
+
+  .social-login {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+
+    .social-button {
+      height: 40px;
+      width: 230px;
+    }
+
+    .google-button {
+      margin-bottom: 10px;
+    }
+
+    .facebook-button {
+      // Blue facebook
+      background-color: #4267B2;
+    }
+
+    .container {
+      display: flex;
+
+      .logo {
+        padding-left: 8px;
+        margin-right: 24px;
+
+        img {
+          width: 18px;
+          height: 18px;
+          vertical-align: middle;
+        }
+      }
+
+      .label {
+        font-family: 'Roboto-Medium', 'Helvetica Neue', 'sans-serif';
```

Comment:
why this class should have a different font-family?

Reply:
https://developers.google.com/identity/branding-guidelines
Styleguide for google social login button say that the font family is Roboto Medium